apps/ledger-live-mobile/src/screens/Settings/Debug/Features/index.tsx

```diff
@@ -9,14 +9,13 @@ import SettingsNavigationScrollView from "../../SettingsNavigationScrollView";
 import { StackNavigatorNavigation } from "~/components/RootNavigator/types/helpers";
 import { SettingsNavigatorStackParamList } from "~/components/RootNavigator/types/SettingsNavigator";
 import { FeatureToggle } from "@ledgerhq/live-common/featureFlags/index";
-import { KeysPriceChange } from "@ledgerhq/live-common/market/utils/types";
 
 export default function Features() {
   const navigation = useNavigation<StackNavigatorNavigation<SettingsNavigatorStackParamList>>();
   const navigateToLargeMover = () => {
     navigation.navigate(ScreenName.LargeMoverLandingPage, {
-      currencyIds: ["bitcoin", "ethereum", "ripple"],
-      initialRange: KeysPriceChange.day,
+      currencyIds: "BTC,ETH,SOL",
```

Comment:
we should enforce this to do it always as uppercase or lowercase (given the signature of the function that uses it)